lib/Dialect/FIRRTL/Transforms/GrandCentral.cpp

```diff
@@ -1081,9 +1081,9 @@ parseAugmentedType(ApplyState &state, DictionaryAttr augmentedType,
     // Sink type must be passive.  It's required to be converted to a NodeOp by
     // the wiring problem solving, and later checked to be a Node.
     // This also ensures passive sink so works equally well w/ or w/o probes.
-    auto sinkType = source->getType();
-    if (auto baseSinkType = type_dyn_cast<FIRRTLBaseType>(sinkType))
-      sinkType = baseSinkType.getPassiveType();
+    auto sinkType = type_cast<FIRRTLBaseType>(source->getType());
```

Comment:
This is stepping up GC to only be allowed to work with base types? Maybe that's fine, but I guess just something to keep in mind.

Reply:
GC can only operate on things which exist in firrtl input, which is limited to firrtl types, so I don't think this really changes anything.

Reply:
Right, FIRRTLType vs FIRRTLBaseType.  Anyway not sure if reachable (view of a probe, property, so on??).  Nevermind.